README.md

```diff
@@ -1,18 +1,692 @@
+
+Add your repository under the `documentation_samples_repos` section of the [Assembly configuration](https://github.com/temporalio/documentation/blob/main/assembly/config.json) file.
+
+Specify a branch name in the `ref` field if your changes are not merged into the main branch of the source repo.
+
+Example:
+
+```go
+{
+  "org": "your-org" // for us that temporalio
+  "name": "your-repo" // name of the repository
+  "ref" : "your-branch-name" // name of your branch - optional
+  // if a branch is not specified, main is used
+}
+```
+
+#### Identify DACX files
+
+The Assembly Workflow identifies DACX files by the inclusion of `_dacx` in filenames.
+In your source repository, add `_dacx` at the end of the file name, but before the file extension, to identify files using DACX.
+
+For example, `my_main_workflow_file_dacx.go`.
+
+This must be done for every file that contains DACX.
+
+#### Write documentation using vanilla Markdown
+
+In the source repository, write your documentation as Markdown using multiline comments.
+
+The Assembly Workflow identifies all multiline comments as Markdown documentation.
+
+Single-line comments are treated the same as the rest of the code.
+
+Single line commenting is still highly encouraged to make the code understandable.
+
+For example:
+
+```go
+/*
+This is treated as Markdown documentation.
+
+Format this information as you would like any other Markdown content.
+
+Since Markdown maintains readablibilty, anyone can read this, even when a [link](/somelink) is present.
+*/
+
+// MyCode is my code and this is treated the same as the code
+func MyCode () error {
+ // ...
+}
+```
+
+#### Use `@dacx` comments to identify info nodes
+
+In the source repository, add `@dacx` to a multiline comment at the bottom of the `_dacx` file.
+
+The comment must be structured exactly like the following:
+
+```go
+/* @dacx
+id: id-of-resulting-info-node
+title: Title of the resulting info node
+label: Info node label (often becomes the anchor if node is used as a header)
+description: Longer description of the info node used in link page previews.
+lines: line numbers
+@dacx */
+```
+
+Example:
+
+```go
+/* @dacx
+id: how-to-run-a-temporal-cloud-worker-in-go
+title: How to run a Temporal Cloud Worker in Go
+label: Run a Cloud Worker
+description: Use a certificate key pair and your Temporal Cloud Namespace to connect to Temporal Cloud.
+lines: 1-52, 64
+@dacx */
+```
+
+#### Basic line selection requirements
+
+You must select both the opening multiline comment line and the closing multiline comment line when selecting multiline comments, as either a distinct selection or as part of a large group of Markdown and code.
+
+For example:
+
+```go
+/*
+This is treated as Markdown documentation.
+
+Format this information as you would like any other Markdown content.
+
+Since Markdown maintains readablibilty, anyone can read this, even when a [link](/somelink) is present.
```

Comment:
```suggestion
Format this information as you would any other Markdown content.

Because Markdown maintains readability, anyone can read this, even when a [link](/somelink) is present.
```